api/src/main/resources/client_api.yaml

```diff
@@ -247,4 +271,4 @@ definitions:
 
   Operator:
     type: string
-    enum: [EQUAL, LESS_THAN, GREATER_THAN, LESS_THAN_OR_EQUAL_TO, GREATER_THAN_OR_EQUAL_TO, LIKE, IN]
+    enum: [EQUAL, NOT_EQUAL, LESS_THAN, GREATER_THAN, LESS_THAN_OR_EQUAL_TO, GREATER_THAN_OR_EQUAL_TO, LIKE, IN]
```

Comment:
So what happens when i combine IN operator with allOf filter?

Reply:
The operator is a field on ColumnFilter, not ResultFilters... you would make a ResultFilter containing a ColumnFilter with IN if you want an in clause on some column.

"IN" can be used on ColumnFilter with values or valueNumbers to make an in clause.